### PR TITLE
Empty state

### PR DESCRIPTION
**Pull request by tetogomez**
### What does this PR do?
- Add Empty State
- Add Meta tag on helmet component
- Add social media preview
- Fix Max width on wide monitor
- Refactor progress bar styling component for liquidity view
- Create EmptyMessage component
- Refactor collapsible table

### Steps to test
1. yarn
1. yarn start
1. go to localhost:3000

#### CheckList
- [ ] Follow proper Markdown format
- [ ] The content is adequate
- [ ] The content is available in both english and spanish
- [ ] I Ran a spell check

### CODE REVIEW

File: src/components/PageTitle/TitlePage.js [hunk 1]
```diff
@@ -5,6 +5,26 @@ import PropTypes from 'prop-types'
 const TitlePage = ({ title }) => (
   <Helmet>
     <title>{title}</title>
+    <meta name="title" content="" />
```
Comment (xavier506) [inline src/components/PageTitle/TitlePage.js:8]: we need to add the right content for these tags

File: src/components/PageTitle/TitlePage.js [hunk 2]
```diff
@@ -5,6 +5,26 @@ import PropTypes from 'prop-types'
...
+    <meta name="description" content="" />
+    {/* Open Graph / Facebook */}
+    <meta property="og:type" content="website" />
+    <meta property="og:url" content="https://metatags.io/" />
```
Comment (xavier506) [inline src/components/PageTitle/TitlePage.js:12]: https://evodex.io

File: src/components/PageTitle/TitlePage.js [hunk 3]
```diff
@@ -5,6 +5,26 @@ import PropTypes from 'prop-types'
...
+    <meta property="og:title" content="" />
+    <meta property="og:description" content="" />
+    <meta
+      property="og:image"
+      content="https://user-images.githubusercontent.com/5632966/92547334-e3e4a600-f211-11ea-864e-a4cfee7ebba8.png"
```
Comment (xavier506) [inline src/components/PageTitle/TitlePage.js:17]: this needs to be the image hosted on our site

File: src/components/PageTitle/TitlePage.js [hunk 4]
```diff
@@ -5,6 +5,26 @@ import PropTypes from 'prop-types'
...
+    />
+    {/* Twitter */}
+    <meta property="twitter:card" content="summary_large_image" />
+    <meta property="twitter:url" content="https://metatags.io/" />
+    <meta property="twitter:title" content="" />
+    <meta property="twitter:description" content="" />
+    <meta
+      property="twitter:image"
+      content="https://user-images.githubusercontent.com/5632966/92547334-e3e4a600-f211-11ea-864e-a4cfee7ebba8.png"
```
Comment (xavier506) [inline src/components/PageTitle/TitlePage.js:26]: this needs to be the image hosted on our site

File: src/components/PageTitle/TitlePage.js (hunk 3 above)
Comment (tetogomez) [reply src/components/PageTitle/TitlePage.js:17]: I'm not sure if it is posible!

File: src/language/en/en.json [hunk 5]
```diff
@@ -6,7 +6,11 @@
     "headerTitle": "Token Listings",
     "version": "Version",
     "footer": "Evodex is open source, get involved.",
-    "placeholder": "This Amount"
+    "placeholder": "This Amount",
```
Comment (xavier506) [inline src/language/en/en.json:9]: @tetogomez I think we should replace "This Amount" for  "Enter Amount"

Comment (tetogomez) [reply src/language/en/en.json:9]: in develop branch it is the current value

File: src/components/InputTextAndSelect/InputTextAndSelect.js [hunk 6]
```diff
@@ -184,15 +184,15 @@ const InputTextAndSelect = ({
           <Select
             id={id}
             onChange={(e) => handleOnChange(e.target.value, 'selectValue')}
-            value={inputData.selectValue || ''}
+            value={inputData.selectValue || t('selected')}
```
Comment (xavier506) [inline src/components/InputTextAndSelect/InputTextAndSelect.js:187]: select

File: src/language/en/en.json [hunk 7]
```diff
@@ -6,7 +6,11 @@
...
+    "selected": "Selected",
```
Comment (xavier506) [inline src/language/en/en.json:10]: Select

**Review by xavier506 — CHANGES_REQUESTED** [review]
Looks Amazing !

please see my comments

**Review by xavier506 — APPROVED** [review]

**Comment by xavier506** [comment]
Nice Job 💯

**Pull request merged**